compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -1163,7 +1196,7 @@ impl<K: DepKind> CurrentDepGraph<K> {
                 record_graph,
                 record_stats,
             )),
-            new_node_to_index: Sharded::new(|| {
+            anon_node_to_index: Sharded::new(|| {
                 FxHashMap::with_capacity_and_hasher(
                     new_node_count_estimate / sharded::SHARDS,
```

Comment:
This size estimate is probably a bit off now, but I'm not sure what we'd replace it with.